services/brig/brig.integration.yaml

```diff
@@ -190,16 +190,10 @@ optSettings:
   setFederationDomain: example.com
   setFeatureFlags:  # see #RefConfigOptions in `/docs/reference`
   setFederationDomainConfigsUpdateFreq: 1
-  setFederationStrategy: allowList
+  setFederationStrategy: allowAll
```

Comment:
probably need to do this in more places?

Reply:
I don't think so. This will flow down to other services calling to brig in the integration tests.